zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -301,6 +302,14 @@ void setNotebookRepo(NotebookRepo repo) {
     this.repo = repo;
   }
 
+  void setRevisionSupported(NotebookRepo repo) {
+    if (repo instanceof NotebookRepoWithVersionControl) {
```

Comment:
IIUC, repo must be `NotebookRepoSync`,  so it would always be NotebookRepoWithVersionControl

Reply:
Have made the relevant changes.